toplevel/group/group.go

```diff
@@ -144,6 +145,10 @@ func (c config) Apply(address string, entriesBytes []byte, dryRun bool, threadPo
 		dryRunOutput(address, toBeWritten, "written")
 		dryRunOutput(address, toBeDeleted, "deleted")
 		dryRunOutput(address, toBeUpdated, "updated")
+
+		desired := getGroupsWithUsernames(desired, users, address)
```

Comment:
Forgot the graphql query already pulls user names so api calls to make entity IDs is unnecessary. Good call